tutorials/hello/hello_client/config/default/config.json

```
@@ -9,7 +9,8 @@
 			"ipc"
 		],
 		"port": 7887,
-		"host": "127.0.0.1"
+		"host": "127.0.0.1",
+		"allowedMethods": []
```

We can allow all the methods by default for testing application
```suggestion
		"allowedMethods": ["*"]
```